resources/assets/components/Signup/index.js

```diff
@@ -262,6 +263,34 @@ class Signup extends React.Component {
     }
   }
 
+  // Delete a signup.
+  deleteSignup(signupId, event) {
+    event.preventDefault();
+    const confirmed = confirm('🚨🔥🚨 Are you sure you want to delete this? 🚨🔥🚨');
+
+    if (confirmed) {
+      // Make API request to Rogue to delete the signup on the backend.
+      const authorizedRequest = new RestApiClient(window.location.origin, {
```

Comment:
This probably makes sense on the `RestApiClient` defined at `this.api`.

Reply:
@DFurnes will this break other things though since we are only using the tokens for the v3 endpoints?

Reply:
Oh, that's a good thought. I thiiiiiiink other endpoint should just ignore the `Authorization` header since they're relying on `Session`, but it's worth double-checking just to make sure!

Reply:
oh you're right ! I just tried with other functions (updating quantity, history modal, deleting post, tagging) and all still works with the change - thanks @DFurnes !